src/Contracts/DataModels.ts

```diff
@@ -210,9 +210,9 @@ export interface QueryMetrics {
 
 export interface Offer {
   id: string;
-  autoscaleMaxThroughput: number;
-  manualThroughput: number;
-  minimumThroughput: number;
+  autoscaleMaxThroughput: number | undefined;

```

Comment:
Sure

Reply:
I believe these are equivalent under strict mode so either is fine:

`autoscaleMaxThroughput?: number`
`autoscaleMaxThroughput: number | undefined`

Reply:
This post seems to suggest [they're not](https://stackoverflow.com/questions/46397280/typescript-whats-the-difference-between-an-optional-field-and-a-union-with-und), but I don't know about strict mode.
If the field is always there I would opt for the union, otherwise for go optional.

Reply:
Ok, I will keep union type here, thanks